zerver/lib/bulk_create.py

```diff
@@ -76,6 +76,7 @@ def bulk_create_streams(realm: Realm,
                     name=name,
                     description=options["description"],
                     invite_only=options["invite_only"],
+                    is_announcement_only=options["is_announcement_only"],
```

Comment:
Explicitly mentioning it every time like the other parameters was chosen here over defaulting this value to `False` if not specified. Do let me know if this needs to be reversed.

Reply:
I think the other way would probably be cleaner (and really, we should change `invite_only` to work that way as well, since it's false for almost all of them too).  Want to redo this as 2 commits (one switching `invite_only` and the other making this a `options.get("is_announcement_only", False)`?